app/controllers/upvotes_controller.rb

```diff
@@ -7,27 +7,34 @@ def create
         review_id: @review.id,
         user_id: current_user.id
       )
-      @review.score += 1
-
-      if @upvote.save && @review.save
-        flash[:notice] = "Upvote created successfully."
-        redirect_to @beach
-      else
-        flash[:notice] = "Upvote not created successfully."
-        render :new
+      respond_to do |format|
+        if @upvote.save
+          format.html { redirect_to :back,
+            notice: "Upvote created successfully." }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 13, 52 should be on its own line.